Regression: "ceil_mode_" is not defined

**Describe the bug**

Get the following error today.
Git blame says it's probably a regression caused by PR #873 
If so, can someone also add the CI coverage?

```
/tmp/scratch/onnxruntime/onnxruntime/core/providers/mkldnn/nn/pool.cc:257:103: error: 'ceil_mode_' was not declared in this scope
   std::vector<int64_t> y_dims = PoolBase::SetOutputSize(x_shape, x_shape[1], &pads, this->dilations_, ceil_mode_);
```

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- ONNX Runtime installed from (source or binary): source
- ONNX Runtime version: master
- Python version: 3.6
- GCC/Compiler version (if compiling from source): gcc 8.2.0

--- a/onnxruntime/core/providers/mkldnn/nn/pool.cc
+++ b/onnxruntime/core/providers/mkldnn/nn/pool.cc
@@ -254,7 +254,7 @@ Status Pool<T, PoolType>::Compute(OpKernelContext* context) const {
     strides.assign(kernel_shape.size(), 1);
   }
 
-  std::vector<int64_t> y_dims = PoolBase::SetOutputSize(x_shape, x_shape[1], &pads, this->dilations_, ceil_mode_);
+  std::vector<int64_t> y_dims = PoolBase::SetOutputSize(x_shape, x_shape[1], &pads, this->dilations_, this->ceil_mode_);
   Tensor* Y = context->Output(0, TensorShape(y_dims));
 
   size_t num_outputs = OpKernel::Node().OutputDefs().size();

